src/test/java/com/cloudbees/jenkins/support/CheckFilterTest.java

```diff
@@ -75,6 +76,14 @@ public class CheckFilterTest {
     @Rule
     public LoggerRule logging = new LoggerRule().record(AsyncResultCache.class, Level.FINER);
 
+    @After
+    public void stopAgents() throws Exception {
```

Comment:
Maybe this is something that should be added to `JenkinsRule`? Compare https://github.com/jenkinsci/jenkins-test-harness/pull/643.

Reply:
No, actually `Jenkins.cleanUp` was already doing this.